PocketKit/Sources/PocketKit/Settings/PremiumStatusView.swift

```diff
@@ -0,0 +1,164 @@
+import SwiftUI
+import Textile
+import MessageUI
+
+struct PremiumStatusView: View {
+    @Environment(\.dismiss)
+    private var dismiss
+    @StateObject
+    var viewModel: PremiumSettingsViewModel
+    @State var result: Result<MFMailComposeResult, Error>?
```

Comment:
any chance we could handle this result inside the view model and, when you instantiate `MailView` it'd take the arguments from the view model as well?